src/components/tabs/AppVTabsItem.vue

```diff
@@ -6,10 +6,11 @@
       justify-between
       items-baseline
       p-1
-      mb-1
+      pl-2
+      pr-2
+      pb-2
```

Comment:
Probably better expressed as `p-2 pr-1`

Reply:
Ah good, we're still wrapping our heads around Tailwind's way to do things. Padding is slightly different on top to better frame the text, since using the same padding on top and bottom results (in our ways) in an apparent imbalance between the top and bottom whitespace.

Reply:
And while trying it out, `p-2` does work best, so let's just use that